project/AppDependencies.scala

```diff
@@ -3,30 +3,30 @@ import sbt.*
 
 object AppDependencies {
 
-  val bootstrapVersion  = "8.5.0"
+  val bootstrapVersion  = "9.5.0"
   val pekkoVersion      = "1.0.2"
-  val mongoVersion      = "1.8.0"
+  val mongoVersion      = "2.2.0"
 
   val compile: Seq[ModuleID] = Seq(
     ws,
-    "org.scala-lang.modules" %% "scala-xml"                  % "2.2.0",
+    "org.scala-lang.modules" %% "scala-xml"                  % "2.3.0",
     "uk.gov.hmrc"            %% "bootstrap-backend-play-30"  % bootstrapVersion,
-    "uk.gov.hmrc"            %% "domain-play-30"             % "9.0.0",
+    "uk.gov.hmrc"            %% "domain-play-30"             % "10.0.0",
     "uk.gov.hmrc"            %% "tabular-data-validator"     % "1.8.0",
-    "commons-codec"           % "commons-codec"              % "1.16.1",
+    "commons-codec"           % "commons-codec"              % "1.17.1",
     "uk.gov.hmrc.mongo"      %% "hmrc-mongo-play-30"         % mongoVersion,
     "org.apache.pekko"       %% "pekko-connectors-csv"       % pekkoVersion
   )
 
   val test: Seq[ModuleID] = Seq(
     "org.apache.pekko"        %% "pekko-connectors-csv"     % pekkoVersion,
     "org.apache.pekko"        %% "pekko-connectors-xml"     % pekkoVersion,
-    "org.apache.pekko"        %% "pekko-testkit"            % pekkoVersion,
+    "org.apache.pekko"        %% "pekko-testkit"            % "1.0.3",
```

Comment:
Why are we updating just the testkit one and not the val for pekkoVersion on line 7

Reply:
pekkoVersion is also used for pekko-connectors-csv  in line 22. Pekko-connectors-csv is not compatible with pekkoVersion 1.0.3 (latest version for pekko-connectors-csv is 1.1.0 and previous version 1.0.2)